package.json

```diff
@@ -25,10 +25,9 @@
     "storybook": "start-storybook -p 9001 -c .storybook",
     "test": "npm run test:lint && npm run test:unit && npm run dataValidate",
     "test:accessibility": "npm run killApp && npm run build && run-p --race start bbcA11y",
-    "test:accessibility:ci": "npm run build && run-p --race start bbcA11y",
     "test:ampValidate": "npm run killApp && npm run build && run-p --race start amp:validate",
+    "test:ci": "npm install cypress bbc-a11y && run-p --race start cypress && run-p --race start bbcA11y",
```

Comment:
This should be updated to `npx install cypress bbc-a11y`, so the installation doesn't update the package.json.